src/Conversion/ONNXToKrnl/Tensor/Compress.cpp

```diff
@@ -146,7 +147,6 @@ struct ONNXCompressOpLowering : public ConversionPattern {
         }
       }
 
-      Value readIndexMemRef = create.mem.alloca(indexMemRefType);
```

Comment:
This `alloca` doesn't appear to be in a loop. Why do we need to move it?

Reply:
You are correct, let me revert that change. Got confused by a smaller-scope iterate. Tx

Reply:
done